src/sick_tim_common.cpp

```diff
@@ -86,7 +86,6 @@ int SickTimCommon::stop_scanner()
 SickTimCommon::~SickTimCommon()
 {
   delete diagnosticPub_;
-  delete parser_;
```

Comment:
Oh, I should have read the whole pull request first before posting my previous comment. From my point it's okay to move the deletion, so we can leave this PR as it is.